set.mm

```diff
@@ -58689,7 +58689,127 @@ singleton of the first member (with no sethood assumptions on ` B ` ).
 
 $(
 =-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=
-  Well-founded induction
+  Well-founded induction (variant)
+=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=
+$)
```

Comment:
"(variant)" can/should be removed, after the other section was renamed.

Reply:
Variant is left there because of the partial order requirement. The standard version of well founded induction (which is still in my mathbox) does not require this. The difference is that the standard version requires the axiom of infinity.